docs/manual/working/scalaGuide/main/sql/slick/PlaySlick.md

```diff
@@ -133,7 +133,7 @@ Injecting a `DatabaseConfig` instance for a different database is also easy. Sim
 
 Of course, you should replace the string `"<db-name>"` with the name of the database's configuration you want to use.
 
-For a full example, have a look at [this sample projet](https://github.com/playframework/play-slick/tree/master/samples/di).
+For a full example, have a look at [this sample project](https://github.com/playframework/play-slick/tree/master/samples/di). TODO 404
```

Comment:
You can point to https://github.com/playframework/play-slick/tree/2.0.0/samples/basic. This version uses Dependency Injection.